benchmarks/benchmarks/stats.py

```diff
@@ -57,44 +57,52 @@ def time_ttest_ind_diff_var(self):
 
 
 class Distribution(Benchmark):
+    # add distributions here
+    dists = ['cauchy', 'gamma', 'beta']
+
     param_names = ['distribution', 'properties']
     params = [
-        ['cauchy', 'gamma', 'beta'],
-        ['pdf', 'cdf', 'rvs', 'fit']
+        dists, ['pdf', 'logpdf', 'cdf', 'logcdf', 'rvs', 'fit', 'sf', 'logsf',
+                'ppf', 'isf', 'moment', 'stats', 'entropy', 'median', 'mean',
+                'var', 'std']
     ]
+    distcont = dict(distcont)
+    # maintain previous benchmarks' values
+    custom_input = {'gamma': [5], 'beta': [5, 3]}
 
     def setup(self, distribution, properties):
-        np.random.seed(12345678)
-        self.x = np.random.rand(100)
+        self.method = getattr(getattr(stats, distribution), properties)
+        x = np.random.rand(100)
```

Comment:
Let's use the `interval` method to get the 99% interval and scale the x values to be linearly spaced on that.